src/Middleware/ErrorCatcher.php

```diff
@@ -99,7 +100,7 @@ private function getRenderer(string $contentType): ?ThrowableRendererInterface
 
     private function getContentType(ServerRequestInterface $request): string
     {
-        $acceptHeaders = preg_split('~\s*,\s*~', $request->getHeaderLine('Accept'), PREG_SPLIT_NO_EMPTY);
+        $acceptHeaders = HeaderHelper::getSortedAcceptTypes($request);
```

Comment:
It is the only usage of the helper so far. What do you think about making some of the methods of the helper private?

Reply:
It may be problematic in terms of testability.

Reply:
Why? Tests should reflect usage overall.

Reply:
> Why? Tests should reflect usage overall.

This is true. The problem is that e.g. testing the `A` and `B` functions that are built into the `C` function is usually tested with fewer tests than the `A`, `B`, and `C` functions are tested separately. Obviously this is wrong, but usually it will be the result.